### DISCUSSION TITLE
Can I default the proxy to enabled and set the default proxy url?

### DISCUSSION THREAD

**Discussion by zwolfinger** [hoppscotch/hoppscotch#1575] [Q&A] [answered]
We want the proxy enabled and the proxy url pre-set for our users.  I used to accomplish this with some nasty sed statements modifying pages/index.vue and pages/settings.vue.  As the code changes, I have to adjust my sed statements and redeploy.  Is there a better way?

**Comment by zwolfinger** [comment, marked as answer]
Oh, I see they are in newstore/settings.ts now.  Changing them there worked.

**Reply by liyasthomas** [reply]
Glad you found it by yourself. Let us know if there's an other issues. If you're self-hosting Hoppscotch for you workplace / company / making any revenue - consider [sponsoring our project - $5 or more](http://github.com/sponsors/hoppscotch) so that we can sustain this project.
